.gitlab/install_script_testing.yml

```diff
@@ -10,7 +10,7 @@ test_install_script:
     - export TESTING_APT_URL=$DEB_TESTING_S3_BUCKET
     - export TESTING_YUM_URL=$RPM_TESTING_S3_BUCKET
     - export TEST_PIPELINE_ID=$CI_PIPELINE_ID
-    - inv pipeline.trigger-child-pipeline --project-name "DataDog/agent-linux-install-script" --git-ref "main" --variables "TESTING_APT_URL,TESTING_YUM_URL,TEST_PIPELINE_ID"
+    - inv pipeline.trigger-child-pipeline --project-name "DataDog/agent-linux-install-script" --git-ref "main" --variables "TESTING_APT_URL,TESTING_YUM_URL,TEST_PIPELINE_ID,DDR_WORKFLOW_ID"
```

Comment:
This does not JUST apply to this file, it applies to every change where you have updated the `trigger-child-pipeline` call: My understanding is that this will error out because the invoke task does not like undefined variables, and this variable will be undefined when not run by conductor

See https://github.com/DataDog/datadog-agent/pull/21690

Reply:
Wouldn't it be easier to adjust the invoke task to not attempt to forward unknown environment variables?

Reply:
Ah I missed your note in the linked PR, and you have a good point. Maybe an additional parameter to conditionally forward variables if they exist could be a good compromise? Or emiting a warning frem the invoke task when an unknown variable is requested for forwarding?

Reply:
I went the warning way for now but I'm open for alternative approches

Reply:
I am happy with changing the behavior, I just wasn't sure if it was intentional or not. I guess no jobs currently fail to startup in this manner, so it shouldn't break the existing pipeline, but for people who are adding a new job it could be a pain to debug

Which could be solved with a warning, so that works!